lib/example_17x_app_web/controllers/thing_controller.ex

```diff
@@ -1,9 +1,22 @@
 defmodule Example17xAppWeb.ThingController do
   use Example17xAppWeb, :controller
 
+  # We use put_view() so we can control the name of the view rendered. It can be used as a plug or inline
+  # in the function (as shown below in the show() function). For more details on put_view() usage look under
+  # the render() documenation at https://hexdocs.pm/phoenix/0.10.0/Phoenix.Controller.html#render/3
+  #
+  plug :put_view, Example17xAppWeb.ThingView
```

Comment:
To use phoenix 1.7.X directly (as part of the approach in this PR), we need to use `put_view()` as otherwise phoenix 1.7.X will expect a name like `Example17xAppWeb.ThingJSON-API` for the view layer module.

_(This is because phoenix will try to initially base the name on the `:"json-api"` format configured earlier)._